run/django/e2e_test_setup.yaml

```diff
@@ -47,124 +47,105 @@ steps:
           -p ${PROJECT_ID} \
           gs://${_STORAGE_BUCKET}"
 
-  - id: "Add Django secrets to Secret Manager"
-    name: "gcr.io/cloud-builders/gcloud"
+  - id: "IAM and Secrets"
+    name: "gcr.io/google.com/cloudsdktool/cloud-sdk"
     entrypoint: "/bin/bash"
     args:
       - "-c"
       - |
-        echo "
-        DATABASE_URL=postgres://${_DB_USER}:${_DB_PASS}@//cloudsql/${_CLOUD_SQL_CONNECTION_NAME}/${_DB_NAME}
+        echo "DATABASE_URL=postgres://${_DB_USER}:${_DB_PASS}@//cloudsql/${_CLOUD_SQL_CONNECTION_NAME}/${_DB_NAME}
         GS_BUCKET_NAME=${_STORAGE_BUCKET}
-        SECRET_KEY=$(cat /dev/urandom | LC_ALL=C tr -dc '[:alpha:]' | fold -w 30 | head -n1)
-        PASSWORD_NAME=${_SECRET_PASSWORD_NAME}" > ${_SECRET_SETTINGS_NAME}
+        SECRET_KEY=$(cat /dev/urandom | LC_ALL=C tr -dc '[:alpha:]' | fold -w 30 | head -n1)" > ${_SECRET_SETTINGS_NAME}
+
+        sa_email=$(gcloud projects list --filter "name=${PROJECT_ID}" --format "value(projectNumber)")-compute@developer.gserviceaccount.com
 
         ./retry.sh "gcloud secrets create ${_SECRET_SETTINGS_NAME} \
           --project $PROJECT_ID \
           --data-file=${_SECRET_SETTINGS_NAME}"
 
-        gcloud secrets add-iam-policy-binding ${_SECRET_SETTINGS_NAME} \
-        --member serviceAccount:$(gcloud projects list --filter "name=${PROJECT_ID}" --format "value(projectNumber)")@cloudbuild.gserviceaccount.com \
+        ./retry.sh "gcloud secrets add-iam-policy-binding ${_SECRET_SETTINGS_NAME} \
+        --member serviceAccount:${sa_email} \
         --role roles/secretmanager.secretAccessor \
-        --project ${PROJECT_ID}
-
-        rm ${_SECRET_SETTINGS_NAME}
-
-        echo -n "${_SECRET_PASSWORD_VALUE}" > ${_SECRET_PASSWORD_NAME}
-
-        ./retry.sh "gcloud secrets create ${_SECRET_PASSWORD_NAME} \
-          --project $PROJECT_ID \
-          --data-file=${_SECRET_PASSWORD_NAME}"
-
-        gcloud secrets add-iam-policy-binding ${_SECRET_PASSWORD_NAME} \
-          --member serviceAccount:$(gcloud projects list --filter "name=${PROJECT_ID}" --format "value(projectNumber)")@cloudbuild.gserviceaccount.com \
-          --role roles/secretmanager.secretAccessor \
-          --project ${PROJECT_ID}
-
-        rm ${_SECRET_PASSWORD_NAME}
+        --project ${PROJECT_ID}"
 
   - id: "Build Container Image"
-    name: "gcr.io/cloud-builders/docker"
-    entrypoint: "/bin/bash"
-    args:
-      - "-c"
-      - |
-        ./retry.sh "docker build -t gcr.io/${PROJECT_ID}/${_SERVICE}:${_VERSION} ."
+    name: gcr.io/k8s-skaffold/pack
```

Comment:
this look like it's deprecated/unsupported

Reply:
It's the one currently listed on https://cloud.google.com/docs/buildpacks/build-application#build_an_application_remotely

If that changes, we should promptly update all samples that uses the old image.

Reply:
I can try the (probable) replacement "buildpacksio/pack", as this seems to work in isolation.